README.md

```diff
@@ -30,7 +30,19 @@ If you have licensed copies of `Basier Circle` and `Gilroy`, you can add them to
 
 ### Setup environment
 
-The development environment currently depends on [Node.js version 16 (lts/gallium)](https://nodejs.org/download/release/latest-v16.x/). The below instructions assume [nvm](https://github.com/nvm-sh/nvm) is being used to manage Node.js versions.
+The development environment currently depends on:
```

Comment:
Here we can directly refer https://github.com/LiskHQ/lisk-sdk/tree/development/sdk#dependencies
As this is dependency of SDK and they might resolve this later point in time, also this PR is created without any issue association